bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+from cowsay import char_names, get_output_string
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(
+        aliases=["cow"],
+        help=f"""
+        Generates a cowsay string and sends it. The available characters for cowsay are:
+
+        {', '.join(char_names)}"""
+    )
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: str = None) -> None:
+        """Builds a cowsay string and sends it to Discord."""
+        if not text:
+            text = f"I'm a {character.lower()}"
+        text = text.lower()
+        character = character.lower()
+        if len(text) >= 150:
+            embed = Embed(
+                title=random.choice(NEGATIVE_REPLIES),
+                description="The given text is too long! Please submit something under 150 characters.",
+                color=Colours.soft_red
+            )
+            await ctx.send(embed=embed)
+            return
```

Comment:
Instead of doing this three times,  you can just do `raise commands.BadArgument(error_message)`, and the error handler will catch it.

Reply:
so, set error message as a constant?

Reply:
No, I mean just do
```py
raise commands.BadArgument("The given text is too long! Please submit something under 150 characters.")
```

Reply:
resolved, will commit soon